### ISSUE TITLE
Add rendering for boundary=protected_area

### ISSUE THREAD

**Issue by daganzdaanda** [gravitystorm/openstreetmap-carto#603]
After #563 is sorted, maybe we can have a look at http://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area and the values of protect_class there. 
If I understand correctly, these aren't rendered at all at the moment. But the tag is used, and sometimes replaces the less specific leisure=nature_reserve.

**Comment by planemad** [comment]
In light of the difficulties implementing rendering of boundary=protected_area fully, can we atleast start recognizing some of the tag combinations to enable the migration of tags from national_park and nature_reserve?

My suggestions is to start rendering [boundary=protected_area][protect_class=2] exactly like [boundary=national_park].

Starting step would be to add the protect_class key so that we can start playing around. Not sure what the procedure for this is?

**Comment by math1985** [comment]
> In light of the difficulties implementing rendering of boundary=protected_area fully, can we atleast start recognizing some of the tag combinations to enable the migration of tags from national_park and nature_reserve?

Unfortunately not, the entire problem is that we currently can't use protect_class as this key is not loaded in the database.

**Comment by planemad** [comment]
@math1985 when do database keys get updated? This will probably be scheduled for sometime right?

**Comment by tilmanb** [comment]
See #1286 . It will be done at some point in the future.

**Comment by ianmcorvidae** [comment]
I don't know if this is the same problem or a different one, but http://www.openstreetmap.org/way/353717755 is tagged with both boundary=protected_area and leisure=nature_reserve; it renders as leisure=nature_reserve at z13 and higher, but z12 and lower the area doesn't appear at all. http://www.openstreetmap.org/way/163135968 by comparison, tagged only with leisure=nature_reserve, renders properly at all levels. Seems like some sort of conflict.

**Comment by jfirebaugh** [comment]
I changed the [recommended tagging](http://wiki.openstreetmap.org/wiki/US_Forest_Service_Data#Tagging) for U.S. National Forests to `boundary=protected_area` and [updated the forests in USFS Region 6](http://www.openstreetmap.org/changeset/36669896) before realizing this would cause them not to be rendered on osm.org. I'm considering reverting to tagging for the renderer with `boundary=national_park` despite widespread agreement that this is incorrect.

Am I correct in interpreting that the consensus here is that `boundary=protected_area` _should_ be rendered, and now it's just a matter of database update cadence?

**Comment by dieterdreist** [comment]
2016-01-19 0:21 GMT+01:00 John Firebaugh <notifications@github.com>:

> Am I correct in interpreting that the consensus here is that
> boundary=protected_area *should* be rendered, and now it's just a matter
> of database update cadence?
>

I'm in favour of rendering it.

**Comment by imagico** [comment]
> Am I correct in interpreting that the consensus here is that boundary=protected_area should be rendered, and now it's just a matter of database update cadence?

I think the only question is if only some types of protect_class should be rendered (which is not possible at the moment) or if it would be an improvement to generally render boundary=protected_area right now in the same styling as leisure=nature_reserve or boundary=national_park.  This is not an easy decision although if the US community can indeed agree on this kind of tagging (which seems appropriate for National Forests) that would be a big step in the right direction which should be supported by this style IMO.

**Comment by dieterdreist** [comment]
2016-01-19 9:57 GMT+01:00 Christoph Hormann <notifications@github.com>:

> boundary=protected_area right now in the same styling as
> leisure=nature_reserve or boundary=national_park. This is not an easy
> decision



yes, rendering all of them in the style of a nature reserve seems strange,
because the tag also includes socially protected areas like military areas
(only 0,3% currently), historic buildings (heritage), special economic
zones etc.
From a practical point of view, the very most actually used values refer to
natural protection, but there are currently 9,7% public land and 3,3%
cultural assets (heritage, conservation).
There are also 519 objects (2,7%) of political protection areas
(reservations, indigenous areas) that would be very important to show
(IMHO).

**Comment by jfirebaugh** [comment]
> if the US community can indeed agree on this kind of tagging (which seems appropriate for National Forests) that would be a big step in the right direction which should be supported by this style IMO.

I believe there is a consensus in the U.S. community that National Forests should be tagged `boundary=protected_area`, `protect_class=6`. It's a bit of a chicken-and-egg situation though because we'd like that to be rendered first.

**Comment by mboeringa** [comment]
I think this needs to wait for a database reload and availability of hstore. @dieterdreist's figures and remarks about different types of protected areas as defined by the **protect_class=x**, clearly show to much noise that would occur if the main purpose of the proposed rendering is to make visible National Forests and similar nature protected areas. This really requires more dedicated rendering via the not yet available protect_class key.

**Comment by math1985** [comment]
>  Am I correct in interpreting that the consensus here is that boundary=protected_area *should* be rendered

Yes, at least for certain protect classes.

>  now it's just a matter of database update cadence?

Yes - although cadence is a bit of an overstatement as no database update was done since this style was done, and there is not really a process in place.

**Comment by astoff** [comment]
> I'm considering reverting to tagging for the renderer with boundary=national_park despite widespread agreement that this is incorrect.

@jfirebaugh The provisional tagging for the renderer we are using in Brazil is  boundary=protected_area, protect_class=*, leisure=nature_reserve.  We are doing this even for indigenous areas, since we want them to show on the map.

**Comment by kennykb** [comment]
I'm coming late to this, because I fell into it when attempting to repair a problematic 2009 import of New York State protected areas. (These include the huge reserves in the Adirondack Park - the largest park in the Lower 48, larger than the State of Massachusetts.) I have been attempting to avoid tagging for the renderer, and trying to follow recommended practice. The result is an extremely disappointing and confusing rendering, which sorely tempts me to go back to telling lies such as "landuse=forest".

I see that this issue has recently celebrated its second birthday. I understand fully the difficulty involved in rerunning osm2pgsql to get it to load the missing keys (or create hstore for the keys that don't have corresponding columns.) I'm fully aware of it, because I maintain my own PostGIS database of just North America, and can readily imagine scaling that to the planet. Nevertheless, now that two years have gone by, and there are still no visible plans for an upgrade to the rendering database that could enable this proposal, would it be fair to close this item with "Won't Fix"? In the alternative, is there anything that those of us who have an interest in this issue, some of whom have pretty fair programming and DBA skills, can do to help? 

I suspect that the underlying issue is that this particular problem - rendering of nature areas - is not by itself important enough to justify a database rebuild. And that's fair enough. In that case, though, I strongly suspect that the rendering database we have is "good enough" and no future rendering issue will ever be important enough. That's actually pretty fair, too. It's unfortunate that such an impasse will, as a practical matter, lead to a certain amount of tagging for the renderer, but that sort of compromise is pretty typical for a mature project. You get to a point where the cost of breaking compatibility exceeds the benefit of a new feature or bug fix.

**Comment by matkoniecz** [comment]
> would it be fair to close this item with "Won't Fix"

Work on that is in progress, see for example #2128

**Comment by glebius** [comment]
What's the current status of the problem? I'm now working on marine protected areas in California, USA and I see that anything I add isn't rendered. Should I switch back to leisure=nature_reserve?

**Comment by dieterdreist** [comment]
sent from a phone

> On 6 Mar 2017, at 07:08, Gleb Smirnoff <notifications@github.com> wrote:
> 
> Should I switch back to leisure=nature_reserve?


you could keep it as a fallback, it isn't wrong

**Comment by kocio-pl** [comment]
Rendering it is now possible (since v4.0.0). Now we need decisions which exactly types should be rendered and how?

**Comment by talllguy** [comment]
I think that the rendering should be based on the [protection classification](http://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area#Resources-protected-area) (tag is protect_class)

Here are some ideas.

protect_class | my thoughts on style
-- | --
1 to 7 | Similar to nature reserve, as these are nature protected classes. I think   that already rendered nature reserves and these can share the same green   transparent border, but perhaps the boundary ones can have a hash, to denote   their status as a bona fide boundary, similar to other boundaries on OSM.
11 to 16, 19 | these are resource protected areas, so maybe something more like quarry where a hash or pictogram pattern is applied over and throughout   the polygon. Tells users don't do something (dig, hunt, fish, swim, drink,   etc.) anywhere inside the boundary
21 to 29 | socially protected - these are mostly covered by other tags in terms of   the thing that is protected inside the boundary, e.g. a church. The boundary   shows the human constructed virtual "fence" around it. Maybe (not   my expertise) a brown thick transparent border would be a appropriate. it   might be a one off consideration.

**Comment by kocio-pl** [comment]
I think it's not enough to just define rendering of protect_class, we should also define what to show when class tags are absent (like protection_object=* or maybe also protection_title=*). Taginfo may be useful to find out which ones are really used.

Brown line (#734a08 - @tourism/@amenity-brown) is already used for zoo and theme park.

**Comment by matkoniecz** [comment]
I would show nature protected classes exactly like current nature reserves and do not render other protect_class, including cases where protect_class is missing.

**Comment by daganzdaanda** [comment]
I agree, let's start with the nature protected classes first. For classes 1-6 we can really just use the nature reserve style. Class 98 seems to be also for bigger areas. 

But class 7 seems to be for smaller local areas, so they should only start to be rendered at maybe z13. I would add protect_class=97 to this rule, it covers the Natura 2000 / FFH areas, of which there are MANY in Europe, see https://en.wikipedia.org/wiki/Natura_2000#Current_status 

> Natura 2000 protects 27,312 sites with terrestrial area 787,606 km2 (around 18 percent of land of the EU countries) and marine area 360,350 km2 in 2017,

If people seriously start mapping these areas, maybe sometime we will have to finetune the rendering again. Natura 2000 areas can range from "a  single church tower" to "large swaths of sea", which means using way_area might possibly be better than a one-size-fits-all cutoff at zoom 13?

**Comment by matkoniecz** [comment]
> using way_area might possibly be better than a one-size-fits-all cutoff at zoom 13?

Yeah, that is why using current rendering for nature reserves will be fine. Note that tiny ones are not rendered earlier.  See for example http://www.openstreetmap.org/#map=18/50.06180/19.86085 and zoom out, it disappears earlier than on z13.

**Comment by kennykb** [comment]
> these are resource protected areas, so maybe something more like quarry where a hash or pictogram pattern is applied over and throughout the polygon. Tells users don't do something (dig, hunt, fish, swim, drink, etc.) anywhere inside the boundary


Fine-tuning of the rendering may be needed in the future, sure. But I don't think that protect_class will be the chief key driving it.

I may be the only person on this thread who's actually mapped resource protected areas. The once I've done are all cases where the land access/use is less restrictive than 'private - no entry without the owner's permission': land that is either held publicly or else protected by conservation easements that allow for public access. 'protect_class=12 protection_object=water' is one I've used a fair amount. It doesn't mean 'don't swim' or 'don't drink' but rather 'don't operate a motor, don't camp, don't build a fire, don't hunt with lead shot (lead-free is ok in many of the areas).' I'm perfectly fine with nature_reserve rendering for these - in fact, I've tagged them with 'leisure=nature_reserve' in addition to the protect_class, so as to get at least some default rendering. ('Nature reserve' encompasses a lot of things.)

There are many other areas around here that are formally resource-protected, but the restrictions are on the landowner: 'don't build a house here', 'don't clearcut', 'don't discharge effluents', ... I haven't tagged any of those, nor do I intend to, since they're ordinarily not observable on the ground. (There are sometimes signs for truckers on major highways giving contact information for spill response teams.)

What interests me more, given that I'm a hiker, is 'access'. I don't care much about the default rendering - rendering it all as 'nature reserve' is fine - but in my own renderings, I want to be able to distinguish between 'open to the public without formalities', 'open to the public but with formaliies' - around here, there are some areas that have mandatory registration on the day of arrival, but are otherwise ordinarily open, 'open to the public with permission - permission ordinarily granted' (do I need to bring my New York watershed access card? Is access restricted in hunting season?), 'access for a fee', and less permissive access (as with areas with limited numbers of permits, advance clearing of dates, and so on). 

The regulatory regimes surrounding these areas are complex, and we really shouldn't be mapping them. As a recreational user, the distinction between protect_classes 1b, 2, 4 and 12 is subtle and doesn't inform 'what may I do?' Locat trail maps tend to tag with regulatory regime, which usually follows land ownership. One series of which I'm fond uses 'green=state forest; tan=watershed management, open to public; pink=watershed management, New York permit required; gray=private conservancy (Nature Conservancy, Open Space Institute, Mohawk and Hudson Land Consetervancy, etc.), public access with restrictions' but that's often not observable on the ground, where the signage simply gives contact information about whom to ask for permission. 

I don't ask OSM to tell me that 'wild forest' in New York means 'dispersed at-large camping allowed at least a quarter-mile from a road, 200 feet from a trail or permanent open water, below 3500 feet elevation, no parties larger than ten or staying longer than three days at a single site.' I'd be deliriously happy to be able to look up with some reliability: 'this area is designated Wild Forest. That area is Nature Conservancy land. That other area is watershed recreation land,' and then do the research myself on what I need to do for a trip. 

+1 on using 'way_area' relative to the map scale. Some of the protected areas are a few city blocks in extent; others are of comparable size to urban counties. 

Do consider the rendering for nested areas.  '1b inside 2' is a common case here - a Wilderness Area that is part of a large park but enjoys stronger protection than the rest of the park.

**Comment by matkoniecz** [comment]
> Do consider the rendering for nested areas. '1b inside 2' is a common case here - a Wilderness Area that is part of a large park but enjoys stronger protection than the rest of the park.

http://www.openstreetmap.org/#map=18/50.06180/19.86085 example is also covering this case

**Comment by kennykb** [comment]
> http://www.openstreetmap.org/#map=18/50.06180/19.86085 example is also covering this case

Yeah. I just deal with it at a larger scale: http://www.openstreetmap.org/#map=10/43.9992/-74.3960

**Comment by daganzdaanda** [comment]
Great, I forgot we already use way_area for the natural reserves! 

I had a look at the other classes. See https://taginfo.openstreetmap.org/keys/protect_class#values for the values that people already use, even when nobody is rendering the tag. 

"Resources-protected-areas"
Things with "class 16, longtime hazard area" could be definitely useful having on the map early. Maybe with a red boundary instead of green?
Classes 11-15 would be of interest, too, as kennykb tells us. (BTW: there is also http://wiki.openstreetmap.org/wiki/DE:Key:hazmat to tag on roads in water protection areas.) IMHO these resource-protecting areas are more of local importance, so they should start at higher zooms (or have a higher way_area-threshold). Maybe they could be shown with a thinner boundary or a slightly different colour than nature reserves.
I would not render catch-all classes like 19.

"Social-protected-areas"
From this group, I really would like to see class 24, Political protection: reservation / indigenous area / aborigine. 
The rest, I don't think we need: 
Class 25 overlaps with military=* tags, so I don't think we need to encourage this. 
I do like adding historic and heritage tags, but I don't use boundary / protect_class for this and I would not want to see classes 22 and 26 on the standard map. 
Class 23, Protection in favor of economics: special economic zones, ... hm, sounds somewhat strange to me. But showing the extent of some free-trade zones might be interesting.
Class 21, community life, seems a bit vague (who designates an protected area like that?) and I imagine we have other tags to describe what's on the ground.
Class 27 seems to be wrong tagging, and 29 is a catch-all again.

One problem for mapping all these boundaries will be to get valid geometry - there are often signs for these protected areas, but mappers will often have to guess the exact extent. No idea if protectedplanet.net would be a valid source?

**Comment by matkoniecz** [comment]
"class 16, longtime hazard area" sounds similar to military=danger_area

> One problem for mapping all these boundaries will be to get valid geometry

Areas are frequently imported from official sources.

**Comment by daganzdaanda** [comment]
> "class 16, longtime hazard area" sounds similar to military=danger_area

Yeah, it is already in use with double tagging, see Chernobyl: http://www.openstreetmap.org/relation/3311547

But of course it may be something non-military like an industrial wasteland... We could use the same style as the military=danger_area, but having a visible difference might be important, too.
(military related issues: #2405, #2670)

**Comment by ljbade** [comment]
I would like to see rendering of these areas.

I recently added an area with class of 12 for a place called Googong Foreshore that is primarily a water catchment reserve but is also managed as a conservation area and for recreation (hiking, fishing).

I was going to use leisure=nature_reserve but the response from the talk-au mailing list was that protected_area is more accurate.

I then discovered this area is not even rendered!

Please add this to the renderer soon.

**Comment by kocio-pl** [comment]
I'm afraid this is not possible now - please look at the discussion on #2830. In short: I have found that even "safe" set of protect_class (1-6) gave too much results in some places, like 777 polygons in Germany. Until somebody make a proper research and testing, it's not ready to be merged.

**Comment by ljbade** [comment]
That's unfortunate. I added the nature reserve tag as a workaround until the protected area is better supported.

**Comment by kocio-pl** [comment]
Feel free to research the subject - I would be happy to join then as a coder.

**Comment by Klaus-Tockloth** [comment]
Do not render leisure=nature_reserve anymore. Render only:

Strict protected areas (access not permitted), use a green hatch pattern (similar to military area):
boundary=protected_area + protect_class=1

National parks:
boundary=protected_area + protect_class=2

Do not render anything else.

**Comment by Klaus-Tockloth** [comment]
> Could you explain what's the reasoning behind it?

Rendering of all protect class areas is very confusing and disturbing. Germany has a lot of these area. E.g. the german state Nordrhein-Westfalen has over 3000 areas (luckily not all tagged in OSM).

![bildschirmfoto 2017-11-29 um 16 08 40](https://user-images.githubusercontent.com/5493128/33381994-d7ac1484-d51f-11e7-89a1-5974d260610c.png)
https://www.openstreetmap.org/#map=12/51.8472/7.6532

> Interesting idea! Could you give some examples?

https://www.openstreetmap.org/relation/2517094
https://www.openstreetmap.org/way/28151799#map=13/52.2897/14.3692

> Are you sure about this is a proper way of tagging national parks?

Yes, according to the german wiki: https://wiki.openstreetmap.org/wiki/DE:Tag:boundary%3Dnational_park
But you are right, boundary=national_park is also valid.

**Comment by datendelphin** [comment]
This seems country dependent. For Switzerland, there is only one park which is a national park. The other dozen or so parks would vanish, which would be rather inconvenient.

**Comment by kocio-pl** [comment]
That's why I asked for research. We can also ask on the Tagging list, because maybe all that's needed is to add new way of tagging and it makes sense for me, because it's much more specific.

But after all we can make our own decisions what to render, since I was not going to render all the protected areas anyway, only a subset of them.

**Comment by adamfranco** [comment]
In [my region](https://www.openstreetmap.org/#map=10/43.7770/-73.3797), (western Vermont & north-east New York, USA), our protected areas and parks don't form contiguous land-masses, but rather are composed of pockets interspersed with private holdings.

Rendering these protected-area boundaries is incredibly important for many map-users as these boundaries help differentiate where one can legally use the land.

**Example 1: Green Mountain National Forest**
The outer-boundary of the region that the Forest service operates has [one relation](https://www.openstreetmap.org/relation/2030450) (`boundary=national_park`). This outer boundary is both published by the forest service and great to view on low-zoom renderings. Unfortunately, it is far from helpful when trying to figure out "can I camp here?" or "are bicycles/motor-vehicles allowed?".

Within this outer-boundary is are additional relations that define the actual parcel boundaries between public and private land and give indications as to what the usage rights are. Here are some examples:

* [Breadloaf Wilderness](https://www.openstreetmap.org/way/116060602#map=12/44.0102/-72.9034) (`boundary=protected_area`, `protected_class=1b`) a parcel that strictly limits access and usage to only foot-traffic and only on designated trails. Camping is only allowed at designated locations. Other examples include the [Joseph Battell Wilderness](https://www.openstreetmap.org/way/116060596#map=13/43.8918/-72.9326) and [Lye Brook Wilderness](https://www.openstreetmap.org/way/116060614#map=13/43.1094/-73.0183).
* [Moosalamoo National Recreation Area](https://www.openstreetmap.org/relation/1610350#map=13/43.9085/-73.0300) (`boundary=protected_area`, `protected_class=5`) is a parcel that is more along the lines of what many think of as a "park" -- it's an area with many hiking trails, biking trails, toilets, camping areas, overlooks, and a host of recreational opportunities. A similar example in this Forest is the [White Rocks National Recreation Area](https://www.openstreetmap.org/way/116060616#map=12/43.3651/-72.8924)
* The rest of the National Forest parcels are handled by [this relation](https://www.openstreetmap.org/relation/1610352) (`boundary=protected_area`, `protected_class=6`). These are managed by the federal government, but allow logging, mining, and other resource extraction (with permits granted) as well as broad public use for hiking, camping, snowmobiling, and other recreation.

**Example 2: Adirondack Park**
The outer-boundary of the enormous Adirondack Park is [this relation](https://www.openstreetmap.org/relation/1695394#map=9/43.9722/-74.3066) (`boundary=national_park`). The outer park boundaries include huge amounts of public land as well as private land, including numerous towns and villages. The the National Forest boundaries, the outer boundary is useful for context, but not for land-use.

Within the outer boundary there are numerous protected-areas that define the actual publicly-owned parcels that have different usage and access restrictions:

* [High Peaks Wilderness](https://www.openstreetmap.org/relation/6360488#map=11/44.1370/-74.0912) (`boundary=protected_area`, `protected_class=1b`) allows only hiking and camping in designated locations. There are many similar areas, including the [Saint Regis Canoe Area](https://www.openstreetmap.org/relation/6360488#map=11/44.1370/-74.0912) and [McKenzie Mountain Wilderness](https://www.openstreetmap.org/relation/6363536#map=13/44.3563/-73.9953) which border villages and towns within the park.
* [Boreas Ponds Tract](https://www.openstreetmap.org/way/481703859#map=13/44.0082/-73.9182) (`boundary=protected_area`, `protected_class=5`) is publicly owned, but has fewer usage restrictions.


Rendering each of these `protected_class`es (1b, 5, and 6) is very important for having a usable map for any sort of outdoor recreation or forestry purposes. When these are rendered I can see where I am and go into the Forest (but not wilderness areas) and cut firewood or a Christmas tree (with permit). If these `protected_class`es weren't rendered I'd have no way of knowing whether I was actually in the National Forest or trespassing on privately-owned land.  

I disagree with the view that rendering a large number of these relations is a problem -- I find it to be a very useful feature of the map. Instead of limiting what is rendered, I think the focus should be to tweak the rendering so that these borders are visible, but maybe a slightly lighter weight or shade than the National Park boundary rendering.

**Comment by kocio-pl** [comment]
Thanks for writing this analysis!

First: this proposition is about rendering `boundary=protected_area`, which is currently **not** rendered - unless tagged as a `leisure=nature_reserve` at the same time.

I was trying with `protected_class=1-6`, even excluding `heritage=*` and `historic=*` (and `leisure=*` to not count nature reserves) and there were too many of them, which makes visual clutter on the map. But maybe that could just be tweaked, because this scheme is more flexible and granular than `leisure=nature_reserve` (which could be simply dropped).

I would start with rendering national parks (with both existing schemes) instead of nature reserves from z8 and show other classes up to 6 later, so they would be visible, yet wouldn't make a noise. I haven't tested it before, because I thought that just filtering classes and sowing them along with reserves would be enough. Now I see we have a chance to avoid the current horrible state of things (I have seen similar pattern in USA in many places) and that sounds promising.

**Comment by kocio-pl** [comment]
General usage stats ([type 4](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area#Nature-protected-areas) is the most popular):

![taghistory 6](https://user-images.githubusercontent.com/5439713/33390501-545a0750-d536-11e7-961f-57b15bbcdc16.png)
![taghistory 7](https://user-images.githubusercontent.com/5439713/33390555-91fec0fa-d536-11e7-9fd9-f1c4c653b2bd.png)

**Comment by adamfranco** [comment]
As Kevin Kenny notes on the [Tagging list](https://lists.openstreetmap.org/pipermail/tagging/2017-November/034089.html), `leisure=nature_reserve` is often used as a work-around for a lack of `boundary=protected_area` rendering, mirroring what is seen by the relative equivalence of the number of nature_reserves and protected_areas. I'm not sure how complex it would be, but ignoring the `leisure=nature_reserve` tag if `boundary=protected_area` with a `protect_class` is present might be a good way to go to allow a smooth transition to this richer tagging scheme.

**Comment by kocio-pl** [comment]
@adamfranco Something happened to your comment - I have it in my mailbox, but it's missing here.

Anyway I tested if nature reserves are really just a "tagging for rendering" and it looks that yes, so we could just drop them: in Poland I've found such 9249 polygons, but when I excluded boundary=* it was only 1492 (16%) and when I also excluded protect_class=*, I'm left with only 56 of them (0.6%).

**Comment by kennykb** [comment]
Please don't forget the more minor protect_classes.
I've tagged a few hundred protect_class=12 areas in southern New York. They have protection_object=water and should be easy to spot. They are parcels of land that New York City has purchased to keep them from being developed, since New York's water supply depends on surface water in the Catskills and Hudson Highlands. Most are open to public recreation (some with advance formalities) and are tagged foot=yes or foot=permit as appropriate. Of course, they're also redundantly tagged with leisure=nature_reserve, since 'nature reserve' comprises a lot of things. Not rendering them will simply keep the "leisure=nature_reserve" hack alive longer.

Also, around here, protect_class=4 is important. New York's state wildlife management areas are also important public recreation resources, and are tagged thus.

If rendering these things "adds too much clutter" then we need to work on the rendering rules, not ignore them.

**Comment by kocio-pl** [comment]
It seems that protection areas are very broad issue, much bigger than I thought - there are a lot of them and they can have different sizes or classes. Therefore we should concentrate on a big picture first and fine tune it later on. My current plan is to move from inefficient  `leisure=nature_reserve` to `boundary=*` scheme and start with national parks on z8+ and the rest (classes 2-6, so 4 would be included) on z9+, probably with class 1 as a hatched green.

> If rendering these things "adds too much clutter" then we need to work on the rendering rules, not ignore them.

Please note that it's a general style, not specific nature or protection related, so we might stop at some point. There might be some special style for rendering all of them, just like openrailwaymap or openseamap. 

> Not rendering them will simply keep the "leisure=nature_reserve" hack alive longer.

It won't, as I plan to drop rendering the old scheme.

**Comment by kennykb** [comment]
@kocio-pl Please see again the comment about class 12. and the other minority classes. They're rare because few of us have started to tag with them YET, but please don't punish us for trying to be specific!

As far as what appears at a given zoom level, maybe base it on the size (land area) of the region? I'd shed no tears if a little area like [Wolf Hollow](http://www.openstreetmap.org/way/429787427) were to stay hidden until z12 or so, but a big one like [Rossman Hill](http://www.openstreetmap.org/relation/6365096) or [Kaaterskill](http://www.openstreetmap.org/relation/6367009) should probably appear sooner. 

Don't confuse protection level with significance. 

I do think these belong in a general style. People relate to parks, and expect to see them on a map. The lands that I've tagged as protected areas are all open to various sorts of public recreation, with various levels and types of restriction. It's "this area is available for hikingcamping/birdwatching", not "this area is boreal forest (Koeppen class Dfc, Anderson class 42)", nor "the owner of this land has secured a tax break by signing a conservation easement".

**Comment by kocio-pl** [comment]
> Don't confuse protection level with significance.

As I said, the problem is complicated (different countries may have different specific), so it's impossible to come with a proper solution from the start. I propose some initial rule, but if you think you have a better one, please be more specific - what zoom level, what class or size etc.

> The lands that I've tagged as protected areas are all open to various sorts of public recreation, with various levels and types of restriction.

We do render a tags for recreation areas - see #2956. You should just add this tag in parallel.

>  please don't punish us for trying to be specific!

So please give us the rules how to show them in a sane way. However not all objects will ever be displayed on a general style and this is not a punishment for railway or water taggers.

**Comment by kennykb** [comment]
> We do render a tags for recreation areas - see #2956. You should just add this tag in parallel.

Wiki says:
>An open green space for general recreation, which often includes formal or informal pitches, nets and so on. 

A forest preserve is hardly an open green space with pitches, nets and so on. recreation_ground is even worse tagging-for-the-renderer than nature_reserve. I have used the tag when it's appropriate, as with [Mount Van Hoevenberg](http://www.openstreetmap.org/relation/6375875).

> So please give us the rules how to show them in a sane way.

As a first approximation, render protect_class'es 12 and 14 like 4. The areas that I've tagged 21 and 22 are not nature reserves and have other appropriate tagging such as 'park', 'recreation ground', 'camping_site', etc.
It might be wise to do a quick check with Overpass and see if there are nature_reserve's about that have other protect_class'es than the ones we've discussed so far.

**Comment by kennykb** [comment]
Oh... also (this is a question, not a criticism) - in the brave new world where protected_area is rendered, but nature_reserve is not, what should I do with a case like [Wolf Hill](http://www.openstreetmap.org/way/339443446)? It's definitely a nature reserve, but it's owned by a [private conservancy](http://mohawkhudson.org/about-us/), rather than a government entity. It's open to the public for hiking, cross-country skiing, birding, and similar activities. I don't know its legal protected status, if any. I think it's a natural candidate for protect_class=7, but from the discussion that I see above, that won't be rendered. Any suggestions?

I realize that I have maybe a couple of dozen examples like this to retag with whatever we decide. I'm willing to retag- but need guidance about the appropriate tagging. I don't think that a mapper who observes a [signed](http://2.bp.blogspot.com/-m2bHXFyQQuY/UuLSTUw6voI/AAAAAAAAC8c/LrwnVT6uNDY/s1600/P1240735.JPG) reserve in the field, should need to know enough of the legalities to assign an IUCN number before being able to add it to the map and have it displayed.

**Comment by kocio-pl** [comment]
> I think it's a natural candidate for protect_class=7, but from the discussion that I see above, that won't be rendered. Any suggestions?

Statistics show that there's only ~1k of them, so rendering them together with 97-99 (and 12, 14) is possible. The 1-6 was an initial take when I was trying to carefully add new schema, but if we think of replacing it, there is bigger potential for rendering classes of protection.

> I don't think that a mapper who observes a signed reserve in the field, should need to know enough of the legalities to assign an IUCN number before being able to add it to the map and have it displayed.

I guess we might also use another option for general rendering, if we could find some sane values:

https://taginfo.openstreetmap.org/keys/?key=protection_title#values

I would however not render just a `boundary=protected_area`, because there are many classes which are rather not related to nature (some of the Resources-protected-area and Social-protected-area).

What's your opinion?

**Comment by kennykb** [comment]
protection_title is rather too free-form. For instance, I don't expect you to parse out 'Watershed Recreation Area' - I think I'm the only mapper who's used that: it's a quirk of local law.

How about a project to review and retag nature_reserve's before pulling the trigger and abandoning rendering on them? (I think they're all at least protect_class=7, if nothing stronger.) I really don't like mechanical edits, so we shouldn't retag wholesale. Nevertheless, we should at least put mappers on notice if we're going to stop rendering something we once did, and give an option (protect_class=unknown, perhaps?) so that they can keep their stuff on the map without having to do legal research.

**Comment by kocio-pl** [comment]
> protection_title is rather too free-form. For instance, I don't expect you to parse out 'Watershed Recreation Area.' I think I'm the only mapper who's used that: it's a quirk of local law.

:+1: 

> Nevertheless, we should at least put mappers on notice if we're going to stop rendering something we once did, and give an option (protect_class=unknown, perhaps?) 

I have written on the Talk list about it, so the people are already aware of the issue:

https://lists.openstreetmap.org/pipermail/talk/2017-November/date.html#79795

I like the idea of making the transition as smooth as possible, that's why we talk about it before we started coding anything. Thank you for your opinions and propositions - we're still not done with the subject, but I'm happy because I see the progress.

**Comment by SK53** [comment]
A couple of important points:

- Nature Reserves are not always protected areas. I would think about 60% of nature reserves in the UK have minimal protection.

- Nature Reserves and Protected Areas do not always share precisely the same boundary. A nature reserve may be land acquired to enable better & more sympathetic management of a protected area, but the nature of land ownership may be that more or less land is bought than the protected area.

- There exist private nature reserves, which fulfil the basic idea of a nature reserve, but the only protection they may possess is the desire of the **current** landowner. One example is Dairy House NR, Capel (http://www.openstreetmap.org/way/44415717). This has no statutory designations (i.e., protected areas).

- In the UK there are 3 types of designation relevant to protecting nature: National Nature Reserves, Local Nature Reserves and Sites of Scientific Interest. These are disjunct concepts (as SSSI can be an NNR or LNR or part of either; most SSSIs are not nature reserves).

Additionally as @SomeoneElse has pointed out on talk, nature reserves are usually visibly signposted, SSSI polygon information is only available with licence tainted geometries, and is rarely accessible through ground-truthing. 

I therefore suggest that any removal of leisure=nature_reserve from the standard rendering does not accord with the actual reality of tagging for these objects. 

Incidentally, typical protections (SAC, RAMSAR, etc) will overlap in multiple ways, so the notion that they can readily be tagged as hierarchical classes is probably misfounded.

**Comment by kocio-pl** [comment]
Thanks, the discussion on the list also adds some points to think about. My questions:

> Nature Reserves are not always protected areas. I would think about 60% of nature reserves in the UK have minimal protection.

"Minimal protection" is still the protection, isn't it?

> Nature Reserves and Protected Areas do not always share precisely the same boundary. A nature reserve may be land acquired to enable better & more sympathetic management of a protected area, but the nature of land ownership may be that more or less land is bought than the protected area.

Any examples? We had a similar problem with forest imports in Poland - the data were rather about property and did not contain the actual forest. Protection boundaries are even harder, because what does "ground truth" mean in such cases?

> There exist private nature reserves, which fulfil the basic idea of a nature reserve, but the only protection they may possess is the desire of the current landowner.

But desire is hard to verify... The same problem: how we can check the "ground truth" then?

> I therefore suggest that any removal of leisure=nature_reserve from the standard rendering does not accord with the actual reality of tagging for these objects.

Could you rephrase your proposition? I didn't understand your it.

**Comment by kennykb** [comment]
> But desire is hard to verify... The same problem: how we can check the "ground truth" then?

Concrete example: [Huyck Preserve](http://www.openstreetmap.org/relation/4836454#).Its preservation is based on the desire f the landowner; in this case, the desire is easy to verify, because the landowner is a [private, non-profit corporation ](https://www.huyckpreserve.org/who-we-are.html) that has published a detailed [management plan](https://www.huyckpreserve.org/uploads/2/4/5/6/24560510/huyck_stewardship_and_management_plan.pdf). There is [signage](http://3.bp.blogspot.com/-Olirj1mT4yE/U9FectSwqdI/AAAAAAAAH1M/xGaYau-1UYA/s1600/DSCF9864.JPG) in the developed areas (the preserve has laboratories and conference facilities) The 20 km or so of trails that are open to the public are all indicated at intervals with a distinctive [marker](https://www.flickr.com/photos/ke9tv/17076109167).

In the management plan, it's explicitly stated that the preserve enjoys no permanent legal protection:
> The Preserve is a land trust. It does not, however, hold any conservation easements. A
conservation easement is a voluntary legal agreement between a landowner and a land trust or
governmental agency that permanently limits uses of the land in order to protect its conservation
value. 

Is this a protected_area? Debatable: all the 'protected area' discussion on the Wiki seems to presume that a national government is the one doing the protection. If it is a protected_area, what protect_class is it? I have absolutely no idea. Is it a nature_reserve? It says it is, on the Web and on the ground.

How did I get the boundaries to put them on the map? I wrote to the landowner, who provided me with a survey (more detailed and complete than the summary on the published [trail map](https://www.huyckpreserve.org/uploads/2/4/5/6/24560510/2016_trail_map.pdf)) and permission to trace it. (I had already mapped the trails with GPS, but they also graciously granted permission to trace the trail map.)

**Comment by kocio-pl** [comment]
Very interesting, thanks! 

It looks like it's easy to verify boundaries and existence of this object, but it's questionable for me it it's really "nature reserve". The name can be misleading - for example: just because you see a moniker "McDonald's restaurant" doesn't mean you will tag it as amenity=restaurant. 

> If it is a protected_area, what protect_class is it? 

Maybe just a general protection? Kind of voluntary=yes would be great to have, so we could tag the details.

**Comment by kennykb** [comment]
> t looks like it's easy to verify boundaries and existence of this object, but it's questionable for me it it's really "nature reserve". The name can be misleading - for example: just because you see a moniker "McDonald's restaurant" doesn't mean you will tag it as amenity=restaurant.

I surely don't know what else to call it. It's been around, devoted to conservation and ecological research, but open to public recreation, for eighty years now, so its _de facto_ protection is quite strong. It gives the appearance of being better preserved than the adjacent Partridge Run State Wildlife Management Area and State Forest, which are government-owned nature reserves. It offers similar recreational opportunities (except that it forbids horseback riding, snowmobiling and hunting, which are permitted on the state land).

 If it looks like a duck, quacks like a duck, waddles like a duck, and swims like a duck...

**Comment by dieterdreist** [comment]
2017-12-01 15:44 GMT+01:00, kocio-pl <notifications@github.com>:
> It looks like it's easy to verify boundaries and existence of this object,
> but it's questionable for me it it's really "nature reserve". The name can
> be misleading


I don't think this is about the name, as Kenny explained, there is
documentation about nature conservation and you can see it on the
ground, seems a perfect fit.

"A nature reserve is a protected area of importance for wildlife,
flora, fauna or features of geological or other special interest,
which is reserved and managed for conservation and to provide special
opportunities for study or research. Nature reserves may be designated
by government institutions in some countries, or by private
landowners, such as charities and research institutions."

Cheers,
Martin

Looks like a shortcoming of the boundary=protected_area scheme, not to
cater for private institutions / landowners

**Comment by kennykb** [comment]
@dieterdreist: "Looks like a shortcoming of the boundary=protected_area scheme, not to cater for private institutions / landowners"

And it took some research to figure out the very fact that I can't figure out the protect_class! Shouldn't an ordinary mapper, unskilled in the legal details, and without time to do extensive research, be able to observe the signs and map, "there's a nature_reserve here" and sketch in approximate boundaries?

[Private](http://www.openstreetmap.org/relation/7083920) [reserves](http://www.openstreetmap.org/way/345643852) [are](http://www.openstreetmap.org/relation/7393760) [pretty](http://www.openstreetmap.org/way/504850219) [common](http://www.openstreetmap.org/relation/1373490) [in](http://www.openstreetmap.org/way/43442363) [this](http://www.openstreetmap.org/way/43436535) part of the world. I concede that the mapping is all over the place. I've included objects for which I've done no research to assign protected_area and protect_class, but all of the parcels that I link to enjoy some sort of private protection.

**Comment by adamfranco** [comment]
Sorry for accidentally deleting my comment earlier. 

Is it possible to have `leisure=nature_reserve` continue to be rendered (maybe somewhat similar to any new rendering for `protect_class=4`) if there is no `boundary=protected_area,protect_class=*` tagging? If there _is_ `boundary=protected_area,protect_class=*` then that rendering would take priority and the `leisure=nature_reserve` could be ignored for rendering purposes until it gets removed from the object (if ever).

I have no idea if this is possible in the renderer code, but it seems like it would satisfy the valid cases of private nature_reserves and unclear classification -- cases where a mapper knows that there is a reserve there, but doesn't have the ability to investigate the proper `protect_class`. This would also prevent objects that really should be tagged with `boundary=protected_area,protect_class=*` from disappearing before they get more detailed tagging.

**Comment by kocio-pl** [comment]
I wrote a summary about tagging and rendering of protected areas in my diary:

http://www.openstreetmap.org/user/kocio/diary/42861

**Comment by kocio-pl** [comment]
It's not entirely clear how protected area relates to nature reserve, however it's more detailed (protect class defined) and `boundary=national_park` seems to be the same as  `boundary=protected_area + protect_class=2` (national parks). 

Therefore I think we should render:
- z8+ - national parks and wilderness areas (important and usually big):
   - boundary=national_park
   - boundary=protected_area + protect_class=1b,2
- z9+ - important natural protected areas (important and classified):
   - boundary=protected_area + protect_class=1,1a(*),3-6
- z10+ - other natural protected areas (classified)
   - boundary=protected_area + protect_class=7, maybe also 12, 14 and 97-99
- z11+ - natural reserves (not classified)
   - leisure=nature_reserve

_(*) Special rendering for 1a (strict nature reserves) - hatched green._


Rendering changes needed:
- boundaries
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/project.mml#L1166
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/admin.mss#L192
- standard labels
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/project.mml#L1871
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/project.mml#L1903
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/project.mml#L2023
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/amenity-points.mss#L1472
- labels along the boundary (only the `project.mml` probably)
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/project.mml#L2261
  - https://github.com/gravitystorm/openstreetmap-carto/blob/54fae2fe7a43b9069b05f8a6c02327abeb70d7d0/admin.mss#L180

**Comment by almccon** [comment]
I don't want to derail the important progress that's now being made about rendering protected natural areas, but I'd also like to suggest (in agreement with @daganzdaanda) that we also include `protect_class=24` to show aboriginal reserves. Probably we'd want to use a color other than green, but I think even just trying it out with green would be better than nothing. 

I'm aware that it was mentioned three years ago in this thread that `boundary=aboriginal_lands` also exists, but it still isn't rendered either. Currently it seems like `protect_class=24` has slightly wider usage, both in terms of numbers and its global coverage.

`protect_class=24` has 596 uses https://taginfo.openstreetmap.org/tags/protect_class=24

`boundary=aboriginal_lands` has 414 uses https://taginfo.openstreetmap.org/tags/boundary=aboriginal_lands

Map of `protect_class=24`
<img width="505" alt="screen shot 2017-12-09 at 9 43 57 pm" src="https://user-images.githubusercontent.com/1212178/33802353-195ade3a-dd2a-11e7-86a8-22e46eb51e3c.png">

Map of `boundary=aboriginal_lands`
<img width="496" alt="screen shot 2017-12-09 at 9 44 29 pm" src="https://user-images.githubusercontent.com/1212178/33802355-2fa92d40-dd2a-11e7-8e3b-fe8a3e8c8973.png">

**Comment by kocio-pl** [comment]
Thanks for the suggestion.

The problem is so complex that I try to stick to the core - adding natural protected areas in a sane way.

Any other objects than natural might be very difficult, because there are many types with not exactly known relations, like:
- resources conservation for important single resources (protect_class=11-19) 
- social conservation for community and architecture (protect_class=21-29)
- protection_object=*
- protection_title=* 

The problems to solve:
- how to choose which ones are interesting for general map (all of them can be rendered on some kind of specialized "protection map")
- how to group them for rendering (dozen of different renderings for each class, object or title is also not suitable for general style)
- how to render them once we have such "clusters" (for example brown border is used for zoos and thematic parks and violet for admin boundaries). 

If you have ideas how to deal with such general issues, it would be helpful. I don't think adding new types one by one is possible if we want to avoid total mess.

**Comment by kocio-pl** [comment]
After some more research it looks like most of my assumptions proved to fail. 

For example some national parks in Poland are smaller than protected landscape parks ( https://commons.wikimedia.org/wiki/File:Parki_narodowe_Polski.png ), so the size is variable and not related to preservation status in any meaningful way. Hence it's hard to make rendering hierarchy (other than just more or less specific tagging).

Another thing that I was pretty sure of, was that national parks have protection class 2. But out of 23 such parks in Poland only 15 have it, but for example at least two of them has class 5 (according to https://pl.wikipedia.org/wiki/Kategorie_ochrony_przyrody_Mi%C4%99dzynarodowej_Unii_Ochrony_Przyrody ). I don't know about other 6, but that's enough to see that there are no hard rules for relations between the local types of protection and IUCN categories.

What assumptions seem to be still true?
1. Every national park and nature reserve are made for protection (even if protection categories can't be mapped 1:1).

What does it mean?
1. Both types of tags should be still used, probably the best if both are applied to every object.
2. We might need more tags for local types (protected landscape parks have no special tagging as far as I know).
3. Key leisure= is still wrong for natural reserves.
4. Rendering may rely only on the size (luckily #2119 is merged already).

**Comment by skylerbunny** [comment]
I'd like to suggest that at some point, a trigger is just going to have to be pulled here. boundary=protected_area has as many relations as leisure=nature_reserve, to name one example. I suspect a lot of those cases (and I'm guilty of them myself) are a matter of of 'tagging for the renderer', in this case, OpenStreetMap Carto.

In the name of moving forward on this, may I suggest:

* Render ways and relations with the tag boundary=protected area as follows:
  * closed ways (meaning areas) are rendered in the same style as leisure=nature_reserve (border names with green outlines and a center label)
  * Relations with closed polygon rings are rendered in the same style as leisure=nature_reserve (border names with green outlines and a center label for each closed ring)
* In cases where leisure=nature_reserve, landuse=forest, boundary=national_park, and leisure=park (all four are ones where osm-carto will render a name and area) already exists on one of the above objects, simply do a check to 'not render it a second time'.

All of the above said, let's not worry about what zoom level they're seen at, which countries do or don't have a certain percentage of these, and so on. I understand the concerns, but we'll be here forever and this will never get executed if we get bogged down in that - we're already four years in, and doing nothing will _never_ make this problem simpler; it will actually make it even more complicated as people continue to use whatever tag they feel best to make sure osm-carto rendering appears and the core problem (double/triple/+ tagging) diverges even more than it already has.

The above is not a perfect solution, and it won't take into account all the edge cases where people are using (yet more) tagging for the renderer, but it at least gets 'what's already out there' rendered _somehow_, and will allow removing these most common redundant and often incorrect tags over time preferentially for 'protected area'. Once some of this cleanup is done, we can move on to differentiating the various render styles for different protected areas by zoom level, type, and so on.

'National parks and state parks look the same' is a problem we can handle if the only difference is 'which protect class they're under'. There are too many if/then/elses with all of the tagging on the board, while also _not_ rendering what people are clearly moving toward.

**Comment by dieterdreist** [comment]
sent from a phone

> On 11. Jun 2018, at 18:18, skylerbunny <notifications@github.com> wrote:
> 
> will allow removing these most common redundant and often incorrect tags over time preferentially for 'protected area'.


if leisure=nature_reserve is incorrectly set on objects that aren’t nature reserves, you are not only allowed to remove them, you should. There is no need to render a different tag in order to solve these problems, still I would generally support rendering at least some of the protected areas.


cheers,
Martin

**Comment by kocio-pl** [comment]
It's doable for some types (IMO related to the nature), especially after we have set a size rule - the only one that makes sense here.

**Comment by kennykb** [comment]
[TL;DR: +1 on using size to decide what to render. -1 on removing nature_reserve tags until a better option for rendering is available. I don't care whether national parks look like state parks. Protect_class is of minimal value to the renderer.] 

Choosing which objects to render based on size is entirely sensible. 

Rendering at least something is a must. Otherwise, you punish mappers for 'doing it right'. Objects like US National Forests are nature reserves only by stretching the term far beyond anything that looks reasonable, but they are too important not to render, and there is a fragile working consensus that nature_reserve as temporary tagging is 'least worst' until the rendering situation improves.

Because of that fragile consensus, removing 'leisure=nature_reserve' tags from these objects before offering a reasonable alternative would be a hostile act. I for one put in a fair amount of work trying to get the protected_area tagging right, and I'd be quite cross if I were to be rewarded for that by seeing New York State lands disappear from the map.

'National parks look the same as state parks' is a non-problem around here. New York's larger state parks are of size comparable to the US National Parks, they have similar usage patterns (including extensive opportunities for backcountry recreation in some very wild land), they are managed by a sovereign entity (US has a principle of dual sovereignty; for the central government to take these over without the state's consent would likely precipitate a constitutional crisis), and some of them are protected more strongly (requiring a constitutional amendment to reclassify them).

The maps that I render are targeted chiefly toward hiking. I find it most useful to derive the rendering style based on access (access=* and foot=*) - those, rather that protection class, are what determines "can I hike here?" (And I still have 'foot=permit' about, because the next question is "can I drive straight to the site or do I need to do paperwork first?" )

Protection class, in fact, falls even beyond 'protection title' when I'm working with special-purpose maps locally. 'Wilderness', 'Wild Forest', 'Primitive Area', 'Canoe Area', 'Forest Preserve' are all class 1b, but have different regulations on where one may camp, whether equestrians and cyclists are permitted, what tools are allowed for trail work (need prearranged permission to possess a chainsaw in Wilderness) and so on. By contrast, there's little difference to the back-country traveler around here between a class 1b Wild Forest and a class 6 State Forest - as long as tree harvesting isn't in progress in the State Forest, the rules are pretty much the same.

These are specialized uses. On the main map, probably the only distinction that would be relevant to most users would be to distinguish class 1 (1a, 1b) from lower classes, and I'd be happy without even that.

**Comment by skylerbunny** [comment]
> It's doable for some types (IMO related to the nature)

https://taginfo.openstreetmap.org/keys/protect_class#values

I'd say if you rendered (1, 1a, 1b, 2, 3, 4, 5, 6, 7), and prioritize that render above other types when they exist on the same object, that's more than enough for now. Even if all 9 were rendered identically (for example how nature_reserve is since they're basically 'nature stuff') this would put 70% of all extant protect_class boundaries on the map.

The remaining 30% is split between lower priority stuff:

- 19, 'There isn't a more specific class for this' natural resource area'
- 27, public land, which may be very broad and might want to be rendered less prominently anyway
- 22+24, which probably want to be rendered in a color other than green, and can be taken up later
- Others which are in the low single digit % of use so far.

**Comment by kocio-pl** [comment]
I guess I can simply reuse this code example now: https://github.com/gravitystorm/openstreetmap-carto/pull/2830#issuecomment-329194886, but with the whole Nature-protected-area range (1-7, 97-99), because now we don't have to classify them - they will be shown on the map as early as they are big enough.

**Comment by sig-pnrnm** [comment]
Hi,
I do not have the level in English, nor on OSM, to understand all the subtleties of this subject.
However, I wanted to point out that we have a problem in France of rendering as regards the "Regional Natural Parks" (PNR).
These are rendered on most proprietary maps, but not on OSM.
Also, it is common for well-intentioned contributors to add the tag "leisure = nature_reserve" to allow the display of these parks.
As a geomatician in one of these parks, I find it hard to convince my colleagues to not use "G**gle maps" where PNRs are rendered.
It would be nice to be able to manage this simple case: the PNR can be identified by this combination of tags:
- `boundary = protected_area`
- `protect_class = 5`
- `protection_title = parc naturel régional`

Thanks for your help !

**Comment by Adamant36** [comment]
In California a lot of people tag protected areas as leisure=park to get them to show up on the map. Some of them got re-tagged to boundary=protected_area recently. Which led to notes and messages by people who thought the places had been deleted when they didn't and then were upset that they were not being rendered anymore. Even though the places are still mapped. I share their feelings. Protected areas in California are major areas. As I am sure is true for ones in other places. They should be rendered.

**Comment by dieterdreist** [comment]
sent from a phone

> On 2. Nov 2018, at 13:47, Adamant36 <notifications@github.com> wrote:
> 
> In California a lot of people tag protected areas as leisure=park to get them to show up on the map. Some of them got re-tagged to boundary=protected_area recently.


leisure =park doesn’t seem right, but natural_reserve is a generic tag for nature protected areas that renders.

**Comment by Adamant36** [comment]
@dieterdreist, yeah leisure=park is totally not right. Which is why they get re-tagged. Although, then people will add the park tag to them again. nature_reserve works sometimes. I don't like the leisure insinuation with it for a protected area though. As a lot of protected areas are either not accessible to the public or have very strict usage rules. I don't think anyone is going to tromp through a protected marsh due to how it is tagged on OSM, but its something I think about. Anyway, I prefer the protected area tag over nature reserve, which is to general imho, because it allows for more specific tagging of the  classification of protection the place has. Maybe boundary=protected_area can at least be rendered the same as nature reserves. I don't see any reason why they couldn't be.

**Comment by dieterdreist** [comment]
right, I agree rendering of boundary=protected_area would be desirable for some protection classes (or maybe for all but with different style), just wanted to point out there is kind of a workaround without stretching the tags beyond their meaning.

**Comment by kocio-pl** [comment]
I still plan to do implement it, since it looks like this is parallel type which is not possible to classify related to natural reserves. I just didn't find the time to return to this particular problem yet - #2830 was first try, but I had to rethink the issue to understand it better.

**Comment by sig-pnrnm** [comment]
Rather than want to manage all categories of "`boundary = protected_area`", the easiest way could be to go step by step, on combinations of tags that do not pose problems.

In French, there is an expression that says "the best is the enemy of good": everything in its time.

So, while "preaching for my parish" (not sure that this expression exists in English), I would suggest that you integrate the combination of tags proposed below to the main rendering: it has no ambiguity!

```
    boundary = protected_area
    protect_class = 5
    protection_title = parc naturel régional
```

For other categories/combinations (or for this one if there is debate), this could be managed in individual issues ?

**Comment by kocio-pl** [comment]
Nobody yet claimed that individual classes are the problem, you are the first to suggest it. Why do you think so? My take is to use the whole section of similar classes.

**Comment by sig-pnrnm** [comment]
This may be due to my bad English level!
I see that the subject seems complex, but I do not understand all the complexity!

**Comment by kocio-pl** [comment]
I took some time to look into the problem and here are some key findings (which evolved through the time):
- https://github.com/gravitystorm/openstreetmap-carto/issues/603#issuecomment-350525709
- https://github.com/gravitystorm/openstreetmap-carto/issues/603#issuecomment-350528217
- https://github.com/gravitystorm/openstreetmap-carto/issues/603#issuecomment-350587768
- https://github.com/gravitystorm/openstreetmap-carto/issues/603#issuecomment-397012934

Of course now I have to recollect things once again, but this is doable. Most of the hard work (research) has been done already.

**Comment by dieterdreist** [comment]
Am Fr., 2. Nov. 2018 um 15:41 Uhr schrieb kocio-pl <notifications@github.com
>:

> Nobody yet claimed that individual classes are the problem, you are the
> first to suggest it. Why do you think so? My take is to use the whole
> section of similar classes.
>


originally this was held back because the subtags were not available at the
time (no hstore), and the main tag lumps toghether all kind of protection
for different reasons (e.g. also cultural protection, heritage, etc.) so to
make sense of the tag in rendering it was suggested to wait until the
information in the subtags became available for this style

**Comment by Bibi56** [comment]
> Nobody yet claimed that individual classes are the problem

As Dieter said, it was not the blocker but in this long thread, it's easy to (mis)understand that some classes may be difficult to render, Sylvain just suggests to start with the easy cases (related to natural protection).

**Comment by kocio-pl** [comment]
As far as i remember I was planning to render metaclass Nature-protected-area (1-7,97-99), and not to render Resources-protected-area (11-19) nor Social-protected-area (21-29).

**Comment by kennykb** [comment]
I have mapped about 400 protect_class=12 areas that are for watershed
protection but offer public access for hunting, fishing and hiking. Before
you pull the trigger, let me know what I need to do with these. They're not
strict wilderness, so 1b isn't right. They're not species-protection, so 4
isn't quite right either. They're protected from essentially all
development, 'sustainable' or otherwise, so 6 doesn't quite fit, either.

12 seemed 'least bad' among the classes (with protection_object=water), but
I might want to reconsider if 12 is not going to render. These areas are a
significant recreational resource,  and many adjoin designated wilderness
areas and offer alternative access to them. (I've crossed a few to approach
less-popular mountaineering routes.)

Anyway, let me know what you want me to do.

**Comment by kocio-pl** [comment]
I don't know. There's a general hint how to assign protection class - in short, it might need searching by name or discussing it in a proper place:

https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area#Protect_classes_for_various_countries

**Comment by sig-pnrnm** [comment]
Sorry to seem in a hurry for this rendering question, but do you have any idea when this change will be operational?
Are there still technical blocking points? (I'm not sure given my low level in English :wink:)
It is true that I can not wait to be able to communicate to my partners on the rendering of the Regional Natural Parks in OSM  :smiley:

**Comment by kocio-pl** [comment]
It is mostly tedious, boring job with the code, which was sketched in #2830. I cannot give you a time frame for this, since I have not too much time for OSM Carto lately. If somebody is willing to make a PR, it would be faster probably.

**Comment by matkoniecz** [comment]
It is likely that this issue will be closed by #3509.

Note that it is not rejection for potential rendering of other subtypes of `boundary=protected_area`. Please open new issues like #3520 to propose other related changes.
